---
title: "Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex."
authors:
  - Pollen AA
  - Nowakowski TJ
  - Shuga J
  - Wang X
  - Leyrat AA
  - Lui JH
  - Li N
  - Szpankowski L
  - Fowler B
  - Chen P
  - Ramalingam N
  - Sun G
  - Thu M
  - Norris M
  - Lebofsky R
  - Toppani D
  - Kemp DW 2nd
  - Wong M
  - Clerkson B
  - Jones BN
  - Wu S
  - Knutsson L
  - Alvarado B
  - Wang J
  - Weaver LS
  - May AP
  - Jones RC
  - Unger MA
  - Kriegstein AR
  - West JA
date: "2014-10-01"
doi: "10.1038/nbt.2967"

# Schedule page publish date (NOT publication's date).
publishDate: "2014-10-01"

# Publication type.
# Use the CSL publication type standard (matching academic package).
# Common types: article-journal, paper-conference, manuscript (preprint), book, chapter, thesis, patent
publication_types: ["article-journal"]

# Publication name and optional abbreviated publication name.
publication: "Nature biotechnology"
publication_short: "Nat Biotechnol"

abstract: "Large-scale surveys of single-cell gene expression have the potential to reveal rare cell populations and lineage relationships but require efficient methods for cell capture and mRNA sequencing. Although cellular barcoding strategies allow parallel sequencing of single cells at ultra-low depths, the limitations of shallow sequencing have not been investigated directly. By capturing 301 single cells from 11 populations using microfluidics and analyzing single-cell transcriptomes across downsampled sequencing depths, we demonstrate that shallow single-cell mRNA sequencing (~50,000 reads per cell) is sufficient for unbiased cell-type classification and biomarker identification. In the developing cortex, we identify diverse cell types, including multiple progenitor and neuronal subtypes, and we identify EGR1 and FOS as previously unreported candidate targets of Notch signaling in human but not mouse radial glia. Our strategy establishes an efficient method for unbiased analysis and comparison of cell populations from heterogeneous tissue by microfluidic single-cell capture and low-coverage sequencing of many cells."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: ""
url_code: ""
url_dataset: ""
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

**PubMed ID:** [25086649](https://pubmed.ncbi.nlm.nih.gov/25086649/)
