---
title: Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and
  activated signaling pathways in developing cerebral cortex
authors:
- Pollen AA
- Nowakowski TJ
- Shuga J
- Wang X
- Leyrat AA
- Lui JH
- Li N
- Szpankowski L
- Fowler B
- Chen P
- Ramalingam N
- Sun G
- Thu M
- Norris M
- Lebofsky R
- Toppani D
- Kemp DW 2nd
- Wong M
- Clerkson B
- Jones BN
- Wu S
- Knutsson L
- Alvarado B
- Wang J
- Weaver LS
- May AP
- Jones RC
- Unger MA
- Kriegstein AR
- West JA
date: '2014-01-01'
publishDate: '2025-12-26T05:29:39.696100Z'
publication_types:
- article-journal
publication: '*Nat Biotechnol*'
doi: nbt.2967 [pii]
abstract: Large-scale surveys of single-cell gene expression have the potential to
  reveal rare cell populations and lineage relationships but require efficient methods
  for cell capture and mRNA sequencing. Although cellular barcoding strategies allow
  parallel sequencing of single cells at ultra-low depths, the limitations of shallow
  sequencing have not been investigated directly. By capturing 301 single cells from
  11 populations using microfluidics and analyzing single-cell transcriptomes across
  downsampled sequencing depths, we demonstrate that shallow single-cell mRNA sequencing
  (~50,000 reads per cell) is sufficient for unbiased cell-type classification and
  biomarker identification. In the developing cortex, we identify diverse cell types,
  including multiple progenitor and neuronal subtypes, and we identify EGR1 and FOS
  as previously unreported candidate targets of Notch signaling in human but not mouse
  radial glia. Our strategy establishes an efficient method for unbiased analysis
  and comparison of cell populations from heterogeneous tissue by microfluidic single-cell
  capture and low-coverage sequencing of many cells.
---
