---
title: "An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder."
authors:
  - Werling DM
  - Brand H
  - An JY
  - Stone MR
  - Zhu L
  - Glessner JT
  - Collins RL
  - Dong S
  - Layer RM
  - Markenscoff-Papadimitriou E
  - Farrell A
  - Schwartz GB
  - Wang HZ
  - Currall BB
  - Zhao X
  - Dea J
  - Duhn C
  - Erdman CA
  - Gilson MC
  - Yadav R
  - Handsaker RE
  - Kashin S
  - Klei L
  - Mandell JD
  - Nowakowski TJ
  - Liu Y
  - Pochareddy S
  - Smith L
  - Walker MF
  - Waterman MJ
  - He X
  - Kriegstein AR
  - Rubenstein JL
  - Sestan N
  - McCarroll SA
  - Neale BM
  - Coon H
  - Willsey AJ
  - Buxbaum JD
  - Daly MJ
  - State MW
  - Quinlan AR
  - Marth GT
  - Roeder K
  - Devlin B
  - Talkowski ME
  - Sanders SJ
date: "2018-04-26"
doi: "10.1038/s41588-018-0107-y"

# Schedule page publish date (NOT publication's date).
publishDate: "2018-04-26"

# Publication type.
# Use the CSL publication type standard (matching academic package).
# Common types: article-journal, paper-conference, manuscript (preprint), book, chapter, thesis, patent
publication_types: ["article-journal"]

# Publication name and optional abbreviated publication name.
publication: "Nature genetics"
publication_short: "Nat Genet"

abstract: "Genomic association studies of common or rare protein-coding variation have established robust statistical approaches to account for multiple testing. Here we present a comparable framework to evaluate rare and de novo noncoding single-nucleotide variants, insertion/deletions, and all classes of structural variation from whole-genome sequencing (WGS). Integrating genomic annotations at the level of nucleotides, genes, and regulatory regions, we define 51,801 annotation categories. Analyses of 519 autism spectrum disorder families did not identify association with any categories after correction for 4,123 effective tests. Without appropriate correction, biologically plausible associations are observed in both cases and controls. Despite excluding previously identified gene-disrupting mutations, coding regions still exhibited the strongest associations. Thus, in autism, the contribution of de novo noncoding variation is probably modest in comparison to that of de novo coding variants. Robust results from future WGS studies will require large cohorts and comprehensive analytical strategies that consider the substantial multiple-testing burden."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: ""
url_code: ""
url_dataset: ""
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---

**PubMed ID:** [29700473](https://pubmed.ncbi.nlm.nih.gov/29700473/)
